src/iframe-video.js

```diff
@@ -63,13 +63,18 @@ export function redispatch(element, event, events) {
 /**
  * @param {!./base-element.BaseElement} video
  * @param {string} src
+ * @param {?string=} sandbox
  * @return {!Element}
  */
-export function createFrameFor(video, src) {
+export function createFrameFor(video, src, sandbox = null) {
```

Comment:
suggestion turn the last argument into a attr dictionary. it is likely we need to ad more like `allow=`, we should actually add `allow=fullscreen autoplay` feature policies already.  they don't do much right now though

Reply:
no, "allow" is the new "allow*". would be `allow="fullscreen autoplay"` but we need `allowfullscreen` anyway for compat.